helm/templates/primary/statefulset-primary-rs.yaml

```diff
@@ -60,17 +74,35 @@ spec:
                   fieldPath: metadata.name
             {{- if .Values.usePassword }}
```

Comment:
Should we add a check for `.Values.config.security.authorization == enabled` also? If usePassword is true, mongodb will start with authorization=enabled irrespective of the value in mongodb config file. This is because of `MONGO_INITDB_ROOT_PASSWORD` and `MONGO_INITDB_ROOT_USERNAME`